docs/alerts/alert-notifications.md

```diff
@@ -191,12 +191,92 @@ The primary purpose of alert rules is to send notifications when triggered. This
             </div>
         </a>
     </div>
+    <div class="mdl-cell mdl-cell--3-col">
+        <a href="/docs/integration/integration/alerts-spikesh-integration/">
+            <div class="demo-card-event mdl-card mdl-shadow--2dp">
+                <div class="flip-card-container">
+                    <div class="flip-card">
+                        <div class="side">
+                            <img src="/docs/images/alerts/notifications/spike.svg" alt="Spike.sh" title="Spike.sh Alerts Integration" style="width: 72px;">
+                        </div>
+                        <div class="side back">
+                            <h5>Spike.sh</h5>Easy Sematext alerts integration with Spike.sh on-call management and escalations platform.
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </a>
+    </div>
+    <div class="mdl-cell mdl-cell--3-col">
+        <a href="/docs/integration/integration/alerts-squadcast-integration/">
+            <div class="demo-card-event mdl-card mdl-shadow--2dp">
+                <div class="flip-card-container">
+                    <div class="flip-card">
+                        <div class="side">
+                            <img src="/docs/images/alerts/notifications/squadcast.svg" alt="Squadcast" title="Squadcast Alerts Integration" style="width: 80px;">
+                        </div>
+                        <div class="side back">
+                            <h5>Squadcast</h5>Sent all the alerts to Squadcast modern incident response and on-call management platform.
```

Comment:
Sent -> Send
all the alerts -> all alerts

Reply:
Thanks, adjusted.